project-koku/koku
# Fixing initial ingest for OCP

## Jira Ticket

[COST-2840](https://issues.redhat.com/browse/COST-2840)

## Description

This change will set start/end dates for OCP summary

## Testing

1. Checkout Branch
2. Restart Koku
3. Add OCP source for current + last month
4. Should see both full months ingested/summarized correctly.

## Notes

...
